src/mixer.cr

```diff
@@ -40,17 +40,15 @@ module SDL
 
     # load long sound file
     def self.load_music(filename, type : Type? = nil)
-      #SDL::RWops.open(filename, "rb") do |rwops|
-      #  if type
-      #    audio = LibMixer.load_mus_type_rw(rwops, type.to_s, 1)
-      #    raise SDL::Error.new("Mix_LoadMUSType_RW") unless audio
-      #  else
-      #    audio = LibMixer.load_mus_rw(rwops, 1)
-      #    raise SDL::Error.new("Mix_LoadMUS_RW") unless audio
-      #  end
-      #  audio
-      #end
-      LibMixer.load_mus(filename)
+      rwops = LibSDL.rw_from_file(filename, "rb")
+      if type
+        audio = LibMixer.load_mus_type_rw(rwops, type.to_s, 1)
+        raise SDL::Error.new("Mix_LoadMUSType_RW") unless audio
+      else
+        audio = LibMixer.load_mus_rw(rwops, 1)
+        raise SDL::Error.new("Mix_LoadMUS_RW") unless audio
+      end
+      audio
```

Comment:
Or `sample = Mixer.load(filename)` maybe? as a factory for `Mixer::Sample` for example.

Then maybe introduce a nice API to play/pause/seek a sample, like `sample.play`, `sample.pause`, ... or something along those lines.